src/Microsoft.SqlTools.ServiceLayer/Localization/sr.strings

```diff
@@ -200,6 +200,8 @@ EditDataScriptFilePathNull = An output filename must be provided
 
 EditDataCommitInProgress = A commit task is in progress. Please wait for completion.
 
+EditDataComputedColumnPlaceholder = <TBD>
```

Comment:
><TBD> [](start = 36, length = 5)

We should probably find a better placeholder than this IMO.  Is this what SSDT or SSMS uses?